server/pom.xml

```diff
@@ -149,12 +149,6 @@
       <version>1.18.24</version>
     </dependency>
 
-    <dependency>
```

Comment:
Have you added this in the plugins module?
@dubdabasoduba

Reply:
Yes, it was already available. So I don't think the server needed it.

Reply:
Right, that makes sense.
Let me test the PR first and then we can merge.

Reply:
I have tested the PR, we have to include this as a dependency in the server module because it fails with ClassNotFoundException.
@dubdabasoduba 
![image](https://user-images.githubusercontent.com/62061579/206926359-a840dd4f-b5d7-4a8e-a044-c12ab6cfb266.png)